diff --git a/node_modules/hdkey/lib/hdkey.js b/node_modules/hdkey/lib/hdkey.js
index 3c28932..438e8bc 100644
--- a/node_modules/hdkey/lib/hdkey.js
+++ b/node_modules/hdkey/lib/hdkey.js
@@ -4,7 +4,7 @@ var crypto = require('crypto')
 var cs = require('coinstring')
 var secp256k1 = require('secp256k1')
 
-var MASTER_SECRET = Buffer.from('Bitcoin seed', 'utf8')
+var MASTER_SECRET = Buffer.from('Nist256p1 seed', 'utf8')
 var HARDENED_OFFSET = 0x80000000
 var LEN = 78
 
